src/components/delegateRegistration/delegateRegistration.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+import MultiStep from '../multiStep';
+import SelectName from './selectName/SelectName';
+import Summary from './summary/summary';
+import Status from './status/status';
+import routes from '../../constants/routes';
+import styles from './delegateRegistration.css';
+
+class DelegateRegistration extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.goBack = this.goBack.bind(this);
+    this.goBackToDelegates = this.goBackToDelegates.bind(this);
+    this.submitDelegateRegistration = this.submitDelegateRegistration.bind(this);
+  }
+
+  // istanbul ignore next
+  goBack() {
+    this.props.history.goBack();
+  }
+
+  // istanbul ignore next
+  goBackToDelegates() {
+    this.props.history.push(routes.delegates.path);
+  }
+
+  // istanbul ignore next
+  submitDelegateRegistration(userInformation) {
+    this.props.delegateRegistered(userInformation);
+  }
```

Comment:
This method can IMO be removed and use directly `props.delegateRegistered` instead of it. 

It's IMO more readable to have the same name for the action (`delegateRegistered`) in this component and in its children.